Fix #1350 viewing apps and streaming at Ubuntu Bionic L4T

gabrielformiga opened the pull request:
This PR fixes opening stream / apps on L4T Bionic as reported at #1350

cgutman commented:
It looks like doing creation this way breaks the parent relationship with the views:
```
00:00:08 - Qt Warning: qrc:/gui/AppView.qml:66: Error: Qt.createQmlObject(): Missing parent object
00:00:08 - Qt Warning: qrc:/gui/CenteredGridView.qml:10: TypeError: Cannot read property 'width' of null
```

gabrielformiga commented:
It's when we go back to PcView from AppView screen, creating objects like that the StackView takes the ownership of the created object and destroys it during stackView.pop(). That warning wasn't logged before because when we push by objects instead of url's, the pop action doesn't destroys the object and also sets the parent back to the originalParent. This warning is not really a problem, but if it that log annoys you I can try to find another solution.

gabrielformiga commented:
@cgutman, Fixed the warnings by moving function calls / property definitions that was depending of parent instance during initialization by moving it to Component.onCompleted. Also fixed a object lifecycle problem when transitioning from QuitSegue to StreamSegue. Everything os ok now, with no more nullpointer warnings. Let me know if there is something else to adjust, thanks.
